app/src/main/java/app/revanced/integrations/youtube/patches/components/LayoutComponentsFilter.java

```diff
@@ -82,7 +82,9 @@ public LayoutComponentsFilter() {
                 "post_base_wrapper",
                 "image_post_root.eml",
```

Comment:
Does this still exist

Reply:
"post_base_wrapper" does this still exist but "image_post_root.eml" doesn't anymore